lib/resources/r4/allergy_intolerance.yaml

```diff
@@ -62,21 +62,7 @@ fields:
   cardinality: 0..*
   type: code
   description: low | high | unable-to-assess
-  url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.criticality
-  binding:
-    description: Assertion about certainty associated with the propensity, or potential risk, of a reaction to the
-                 identified substance (including pharmaceutical product).
-    terminology:
-    - display: AllergyIntoleranceCriticality
-      system: http://hl7.org/fhir/allergy-intolerance-criticality
-      info_link: http://hl7.org/fhir/r4/valueset-allergy-intolerance-criticality.html
-
-- name: criticality
-  required: 'No'
-  cardinality: 0..*
-  type: code
-  description: low | high | unable-to-assess
-  url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.criticality
+  url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.criticality
   binding:
     description: Assertion about certainty associated with the propensity, or potential risk, of a reaction to the
                  identified substance (including pharmaceutical product).
```

Comment:
The info_link below must a secured one.
info_link: https://hl7.org/fhir/r4/valueset-allergy-intolerance-criticality.html

Reply:
done